src/Raven.Server/Documents/Transformers/Transformer.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.IO;
+using Raven.Abstractions;
+using Raven.Abstractions.Indexing;
+using Raven.Abstractions.Logging;
+using Raven.Imports.Newtonsoft.Json;
+using Raven.Server.Config.Categories;
+using Raven.Server.Documents.Indexes.Static;
+using Sparrow;
+using Voron.Platform.Posix;
+
+namespace Raven.Server.Documents.Transformers
+{
+    public class Transformer
+    {
+        public const string FileExtension = ".transformer";
+
+        private static readonly ILog Log = LogManager.GetLogger(typeof(Transformer));
```

Comment:
Let us use `LoggerSetup` instead. We'll need to migrate anyway, and it  make sense to do that in new code